src/nectarchain/dqm/mean_camera_display.py

```diff
@@ -28,37 +28,48 @@ def ConfigureForRun(self, path, Chan, Samp, Reader1):
         self.cmap = "gnuplot2"
         self.cmap2 = "gnuplot2"
 
+        self.CameraAverage = []
+        self.CameraAverage_ped = []
+
+
     def ProcessEvent(self, evt, noped):
+        self.pixelBAD = evt.mon.tel[0].pixel_status.hardware_failing_pixels
         pixel = evt.nectarcam.tel[0].svc.pixel_ids
-        pixel21 = np.arange(0,21,1,dtype = int)
-        pixel = list(pixel)
-        pixel21 = list(pixel21)
-        pixels = np.concatenate([pixel21,pixel])
+        if len(pixel) < self.Chan:
```

Comment:
Do not understand the relationship to ctapipe 0.19. Moreover this looks like duplicated code. Should it be merged in a single utility method?

Reply:
This is actually part of #63 and should not be duplicated here. @hashkar , I will remove this part.